tests/at_tests.cpp

```diff
@@ -26,39 +26,39 @@ SUITE(at_tests)
 {
     TEST(static_array)
     {
-        int a[] = { 1, 2, 3, 4 };
+        size_t a[] = { 1, 2, 3, 4 };
 
-        for (int i = 0; i < 4; ++i)
+        for (size_t i = 0; i < 4; ++i)
             CHECK(at(a, i) == i+1);
```

Comment:
I'll test with Clang, but I'm almost positive it will complain about the same thing as I have had to make a lot of changes like this to support Clang Tidy with these warnings enabled